awscli/customizations/wizard/wizards/dynamodb/new-table.yml

```diff
@@ -95,6 +100,101 @@ plan:
         type: prompt
         description: Choose a customer managed CMK
         choices: existing_cmks
+        condition:
+          variable: encryption_settings
+          equals: customer_managed_cmk
+  preview:
+    shortname: Preview
+    description: Preview results
+    values:
+      preview_cli_command_value:
+        type: template
+        value: |
+          aws dynamodb create-table \
```

Comment:
Running through the template, I ran into a condition when a backslash is ended to the last line:
```
aws dynamodb create-table \ 
  --table-name 'MyTable' \ 
  --attribute-definitions \ 
    'AttributeName=ds,AttributeType=S' \ 
  --key-schema \ 
    'AttributeName=ds,KeyType=HASH' \ 
```
It seems like it happens when you do not specify a KMS key.